health/health.d/dbengine.conf

```
@@ -59,6 +62,7 @@ component: DB engine
     every: 10s
      crit: $this != 0
     delay: down 1h multiplier 1.5 max 3h
-     info: number of pages deleted due to failure to flush data to disk in the last 10 minutes. \
-           Metric data were lost to unblock data collection. To fix, reduce disk load or use faster disks.
+     info: Number of pages deleted due to failure to flush data to disk in the last 10 minutes. \
+           Metric data were lost to unblock data collection. \
```

Same as the above. I’d also say ‘dropped’ instead of ‘lost’ as it indicates better that this was intentional.

```suggestion
           Some metric data was dropped to unblock data collection. \
```